js/stores/eventStore.js

```diff
@@ -98,8 +98,10 @@ const doAction = (action) => {
       eventState = eventState.set('page_load', eventState.get('page_load').push(pageLoadEvent))
       break
     case WindowConstants.WINDOW_SET_FOCUSED_FRAME:
-      addPageView(action.frameProps.get('location'))
-      lastActiveTabId = action.frameProps.get('tabId')
+      if (action.frameProps) {
```

Comment:
this looks like a good fix, but I'm thinking that we should apply it in windowActions so anything else that cares about this event in the future won't have the same problem